### PR TITLE
Add SearchBar for layers selection

### PR DESCRIPTION
**Pull request by Jatana**
ezgif-2-c00195b7c555

### CODE REVIEW

File: ide/static/css/searchbar_style.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	transition: 0.3s;
+}
+
+.layer-search-input-selected {
+	opacity: 1 !important;
+}
+
+.matched-search-char {
+	color: rgb(69, 80, 97);
+	// text-decoration: underline;
```
Comment (Ram81) [inline ide/static/css/searchbar_style.css]: If we dont need this please remove it.

File: ide/static/css/searchbar_style.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+#layer-search-input {
+	position: absolute;
+	top: 36px;
+	left: 20px;
+	font-size: 15px;
+	background: none;
+	border: none;
+	color: rgb(69, 80, 97);
+	// border-bottom: 1px solid black;
```
Comment (Ram81) [inline ide/static/css/searchbar_style.css]: If we dont need this please remove it.

File: ide/static/js/content.js [hunk 3]
```diff
@@ -1275,7 +1275,9 @@ class Content extends React.Component {
              />
              <h5 className="sidebar-heading">LOGIN</h5>
              <Login setUserId={this.setUserId} setUserName={this.setUserName}></Login>
-             <h5 className="sidebar-heading">INSERT LAYER</h5>
+             <h5 className="sidebar-heading insert-layer-title"><input id="layer-search-input" placeholder="Search for layer"></input><div id="insert-layer-sign">INSERT LAYER</div><i className="material-icons" id="layer-search-icon">
```
Comment (Ram81) [inline ide/static/js/content.js]: Align `<div>` and `<input>` tags, I mean add indentation and new lines to separate tags

**Comment by yashdusing** [comment]
Looks good to me but I’ve encountered one bug. Whenever I type in a single letter (for eg : ‘r’) , many of the queries return false and it shows no layers as a result. I’ll add in a screenshot in a few minutes so please look into it. Overall, good work though !

**Comment by Ram81** [comment]
@yashdusing Nice catch

**Comment by gautamjajoo** [comment]
@thatbrguy @Ram81 Are we allowed to work on two tasks at a single time because I have seen one more [PR](https://github.com/Cloud-CV/Fabrik/pull/394) by @Jatana which are two differnet tasks. Just a clarification. Thanks! :)

**Comment by thatbrguy** [comment]
@gautamjajoo The task pertinent to PR #394 was given approved by @yashdusing and hence the student is working on this new task. No worries haha

File: ide/static/js/pane.js [hunk 4]
```diff
@@ -26,7 +26,85 @@ class Pane extends React.Component {
...
+              } else {
+                final += sub[i];
+              }
+            }
+            elem.innerHTML = final;
+          } else {
+            elem.style.display = 'none';
+          }
+        }
+        for (let elem of $('.panel-heading')) {
```
[the remark this answers is not in the document]
Comment (Ram81) [reply ide/static/js/pane.js]: @thatbrguy `elem` is being used

**Pull request merged**